commons/commands.py

```diff
@@ -16,7 +16,7 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 """All the constants are alphabetically arranged."""
-CREATE_FILE = "dd if={} of={} bs={} count={}"
+CREATE_FILE = "dd if={} of={} bs={} count={} iflag=fullblock"
```

Comment:
hope this is non-hurting option added to command as this command might be used at various places

Reply:
yup this will not cause any issue. It will help to avoid the partial read warnings in dd commands which leads to failure in m0 utilities.

Reply:
was checking if any other tests other m0 utilities will be affected.

Reply:
I have run others motr tests to verify the same.